pkg/envoy/proxy.go

```diff
@@ -191,6 +191,7 @@ func (e *envoy) Run(abort <-chan error) error {
 		log.Warnf("Aborting proxy")
 		if errKill := cmd.Process.Kill(); errKill != nil {
 			log.Warnf("killing proxy caused an error %v", errKill)
+			return errKill
```

Comment:
It is intended to return the abort err

Reply:
> It is intended to return the abort err

Yes, you are right. Here it is indeed intended to return an abort error. But I think that if other errors are generated during the abort process, I think that error should also be returned to the user at the end, in order to provide users with more complete and accurate error information, because this errKill does not return if an error occurs, just The error message "proxy aborted" is always returned. In fact, at the user level, the "cmd.Process.Kill()" operation may not know that an exception has occurred in the "cmd.Process.Kill()" operation. This may confuse the user's troubleshooting and understanding of termination. think again, thanks

Reply:
Why? 

Before this PR: we log the Kill() error (L193) and bubble up the primary error which is also logged
After this PR: we only show the Kill() error, lose the real error

it is a net loss.